app/domain/agents/gemini_agent.py

```
@@ -0,0 +1,241 @@
+import google.generativeai as genai
+import google.generativeai.types as genai_types
+from typing import Optional, Dict, List, Any, AsyncIterator
```

Ruff (0.11.9)

`typing.AsyncIterator` imported but unused

Remove unused import: `typing.AsyncIterator`

(F401)